### ISSUE TITLE
[Feature] Add kubectl plugin to help scale ray raycluster

### ISSUE THREAD

**Issue by chenk008** [ray-project/kuberay#110]
### Search before asking

- [X] I had searched in the [issues](https://github.com/ray-project/kuberay/issues) and found no similar feature requirement.

### Description

When we want to scale out/shrink a ray cluster, we use `kubectl edit raycluster xxx` to edit the raycluster. Especially in shrinking, we have to add `ScaleStrategy` in the CRD. It is a little complicated.

We can create a kubectl plugin to simplify the way.
```
kubectl ray scale $groupName $replica
kubectl ray shrink $groupName $delete-podname-0 $delete-podname-1
```

@Jeffwan @akanso  WDYT?

### Use case

A kubectl plugin to help user to scale out/shrink easily.

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

**Comment by akanso** [comment]
yes good idea. I guess we would also need to specify the CR instance name for the raycluster.

kubectl ray scale `$raycluster` $groupName $replica

**Comment by Jeffwan** [comment]
`kubectl ray scale $raycluster $groupName $replica` I assume this would be used for both scale out and scale in? 
and shrink is used for fine grain control?

**Comment by akanso** [comment]
I guess scaling down might require an optional flag, which is the name of the pod to scale down. Because in certain situations we cannot take down a random pod, it needs to be an idle one.

**Comment by daikeshi** [comment]
I'm wonder if this functionality can be included in Kuberay CLI? So users don't need to rely on `kubectl` (another CLI) to interact with the Ray clusters.

**Comment by andrewsykim** [comment]
I didn't know this issue existed, but here's a proposal on this idea https://docs.google.com/document/d/1msUpb59QBE9ypc1fLy7Od2FsZOzgRgMkPOJG7xHN1UU/edit?usp=sharing&resourcekey=0-G216URZBuFHOjvgcznRaAQ

@kevin85421 and I discussed this earlier this week. I plan to put together a PR for the initial version soon

**Comment by davidxia** [comment]
Is anyone working on this? Happy to contribute a PR if the CLI interface has been decided.
